.github/workflows/dependencies-macos.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+brew install open-mpi autoconf automake libtool ossp-uuid
+
+# libfabric
+wget https://github.com/ofiwg/libfabric/archive/refs/tags/v1.12.1.tar.gz
```

Comment:
We can upgrade libfabric to at least v1.15.2

Reply:
Done!